app/views/quotes/new.html.erb

```diff
@@ -1,6 +1,12 @@
 <%= form_for @quote do |f| %>
   <div class="row">
     <div class="col-md-6 col-md-offset-3">
+      <% if current_company.credits < 0 %>
```

Comment:
Will the company's credits ever be negative? Maybe here we can check if a company has 0 credits, then display a message saying they need to buy credits in order to submit a quote, and add a button like 'Buy Credits' next to it